.gitignore

```diff
@@ -15,6 +15,12 @@ libs/contentful-extensions/parent-slug/.cache
 
 # dependencies
 node_modules/
+.yarn/*
+!.yarn/patches
+!.yarn/releases
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Why are we committing yarn releases and more in the monorepo?

Reply:
`yarn/releases` contains the executable for yarn, the other ones I don't know so much, but I took this from here:
https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored